haystack/pipelines/base.py

```diff
@@ -684,6 +684,8 @@ def eval(
         documents: Optional[List[List[Document]]] = None,
         params: Optional[dict] = None,
         sas_model_name_or_path: str = None,
+        sas_batch_size: int = 32,
+        sas_use_gpu: bool = True,
```

Comment:
Please add some docstrings.

Reply:
Done. Let me know if these are sufficient or if I should adjust them.

Reply:
I streamlined the docstrings with how we handle comparable params within the code base.